packages/elements-dev-portal/src/containers/StoplightProject.tsx

```diff
@@ -129,7 +130,7 @@ const StoplightProjectImpl: React.FC<StoplightProjectProps> = ({
     elem = (
       <NodeContent
         node={node}
-        Link={Link}
+        Link={ReactRouterMarkdownLink!}
```

Comment:
The reason you had to force the typing here with `!` is because `CustomLinkComponent` is expecting `to` but `ReactRouterMarkdownLink` is expecting `href`

![image](https://user-images.githubusercontent.com/7423098/150859881-43d770d4-e566-436c-a8f8-0ea476e35dc9.png)


![image](https://user-images.githubusercontent.com/7423098/150859954-cfac7bea-9fe3-4e09-a59e-4dd92de75e47.png)